TestResultSummaryService/parsers/Test.js

```diff
@@ -22,6 +22,20 @@ class Test extends Parser {
     }
 
     async parse(output) {
+        if (!output) {
+            return {
+                tests: [
+                    {
+                        testName: 'Pre Test',
+                        testOutput: '',
+                        testResult: 'FAILED',
```

Comment:
The strings 'Pre Test' and 'FAILED' (+ 'Post Test' and 'PASSED') are used at several places in this file. With this new addition it would be great of we can create constants for that strings.